mmdet/structures/bbox/bbox_mode.py

```diff
@@ -0,0 +1,267 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import Callable, Optional, Tuple, Type, Union
+
+import numpy as np
+import torch
+from torch import Tensor
+
+from mmdet.structures.mask.structures import BitmapMasks, PolygonMasks
+from .base_bbox import BaseBoxes
+
+BoxType = Union[np.ndarray, Tensor, BaseBoxes]
+MaskType = Union[BitmapMasks, PolygonMasks]
+
+bbox_modes: dict = {}
+_bbox_mode_to_name: dict = {}
+bbox_mode_converters: dict = {}
+
+
+def _register_bbox_mode(name: str,
```

Comment:
for bbox mode, we usually refer to `xywh` and `x1y1x2y2`.  We could simply use register_box_type of register_box.